### PR TITLE
Matthew embed image update

### PR DESCRIPTION
**Pull request by MatthewHiscock**
Three small changes: 2 images and some code. @mariahhamel @mkcor @chriddyp  @NicoleGrondin  @aneda

### CODE REVIEW

**Comment by mkcor** [comment]
Really? Why switching from PNG to JPEG?

**Comment by MatthewHiscock** [comment]
@mkcor - fixed and ready for review.

File: _posts/2015-04-23-embed-plotly-graphs.md [hunk 1]
```diff
@@ -32,14 +32,14 @@ Notice the sharing options located in the bottom right corner.  You want to clic
...
-This will bring up the **Share Plot** popover. As with option 2, click on the **Embed** &lt;/&gt; button. You’ll need to make sure that either the graph is set to “public” OR the **Enable private share link** option under **Sharable Link** is checked in order to generate the HTML code. | ![Sharable link](/static/images/embed-plotly-graphs/sharable-link.jpg) ![Share plot](/static/images/embed-plotly-graphs/share-plot.jpg)
+This will bring up the **Share Plot** popover. As with option 2, click on the **Embed** &lt;/&gt; button. You’ll need to make sure that either the graph is set to “public” OR the **Enable private share link** option under **Sharable Link** is checked in order to generate the HTML code. | ![Sharable link](/static/images/embed-plotly-graphs/sharable-link.png) ![Share plot](/static/images/embed-plotly-graphs/share-plot.png)
 
-![Final embedding code](/static/images/embed-plotly-graphs/embed-code.png)
+![Final embedding code](/static/images/embed-plotly-graphs/embed-code.jpg)
```
Comment (mkcor) [inline _posts/2015-04-23-embed-plotly-graphs.md]: Not sure about this one!

**Comment by MatthewHiscock** [comment]
Should be ready now! @mkcor

**Comment by mkcor** [comment]
@MatthewHiscock Could you please remove file `static/images/embed-plotly-graphs/embed-code.jpg`?

**Comment by MatthewHiscock** [comment]
@mkcor all jpgs should now be deleted, though I haven't done that before so I might have the process wrong.

**Comment by mkcor** [comment]
Looking good!

Awesome. Thank you for doing all these updates, that's super valuable.

:dancer:

**Pull request merged**